packages/react-components/react-virtualizer/src/components/Virtualizer/Virtualizer.types.ts

```diff
@@ -0,0 +1,128 @@
+import type { ReactNode, MutableRefObject } from 'react';
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type VirtualizerSlots = {
+  before: NonNullable<Slot<'div', 'td'>>;
+  beforeContainer: NonNullable<Slot<'div', 'tr'>>;
+  after: NonNullable<Slot<'div', 'td'>>;
+  afterContainer: NonNullable<Slot<'div', 'tr'>>;
```

Comment:
With the vague naming of these slots, I'd add some documentation as to what they do